src/components/TweetList.vue

```diff
@@ -0,0 +1,18 @@
+<template lang="pug">
+    b-jumbotron
+        b-list-group
+            b-list-group-item(v-for="(node, index) in tweets" :key="index").flex-column.align.items.start
+                .d-flex.w-100.justify-content-between
+                h5.mb-1 {{node['user']}}
+                small 3 days ago
```

Comment:
This was changed in fe8da89dbb31d886f2f3909ab113633c5f348d09 and should be updated here.

Reply:
- fixed fd74b218ad0d36adbc9ab08a663495dee5a2fb62